v2/samples/azureiotedge-simulated-temperature-sensor/ControlCommand.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace azureiotedge_simulated_temperature_sensor
```

Comment:
I see the V1 code has the same patter, but I would prefer to use the standard C# style namespace - Azure.Iot.Edge.SimulatedTemperatureSensor. But I will check with Mike on this.

Reply:
Changed this to `AzureIotEdgeSimulatedTemperatureSensor`